pytest_easyread.py

```diff
@@ -129,18 +146,29 @@ def pytest_runtest_logreport(self, report):
             self._tw.write(" " + line)
             self.currentfspath = -2
 
-    # Reporting failures
-    def _get_failure_title(self, rep):
-        if len(rep.nodeid.split("::")) >= 3:
-            return rep.nodeid.split("::")[1] + ": " + self._get_formatted_test_title(rep)
+    # Functions for reporting failures live below this point
+    def _get_failure_title(self, report):
+        """
+        :param report: report for a failing test (type: object)
+        :return: formatted title of a failing test prefixed with class name if any (type: string)
+        """
+        if len(report.nodeid.split("::")) >= 3:
+            return report.nodeid.split("::")[1] + ": " + self._get_formatted_test_title(report)
         else:
-             return self._get_formatted_test_title(rep)
-
-    # based on `sep` method of the TerminalWriter's class
-    def _ljust_sep(self, rep, sepchar, title=None, fullwidth=None, **kw):
+             return self._get_formatted_test_title(report)
+
+    def _ljust_sep(self, sepchar, title=None, **kw):
```

Comment:
Small point but I'd stick to one of either `**kw` or `**kwargs` for consistency.

Reply:
Good idea, it's not consistent because it's that way in original easy pytest code which I modified, but I can make it better ^_^